src/core/payoutCalculator/PayoutCalculatorIsolateSuperCharge.ts

```diff
@@ -41,7 +43,6 @@ export class PayoutCalculatorIsolateSuperCharge implements IPayoutCalculator {
                     [key: string]: { npsStake: number; commonStake: number; superchargedStake: number };
                 } = {};
 
-                const transactionFees = block.usercommandtransactionfees || 0;
```

Comment:
not related to this change, but removed since inherited from the original payoutCalculator but no longer used (including prior to this change.)